### DISCUSSION TITLE
Remote server woes

### DISCUSSION THREAD

**Discussion by cori** [kantord/SeaGOAT#269] [General]
I'm trying to run `seagoat-server` on a remote host - i've got it running and exposed and have my client configured correctly and when i run the server not in the background I can see my query hit the server, but my query ends up with a `FileNotFoundError` referencing a path that most definitely exists on the remote repo and a file that exists in the local repo at a different path - the two repos are structurally the same of course but at different parent paths.

If I `ln -s` the repo to the same parent on the server side and then run the server from the symlink path, I don't receive an error.

Not sure if this is expected and if so whether this workaround should be documented somewhere, or if respecting a repo root would be more effective.

**Comment by kantord** [comment]
This is almost definitely a bug, could you please report it as an issue? It will get more attention then I think https://github.com/kantord/SeaGOAT/issues/new
